solver/src/main/java/org/chocosolver/solver/expression/continuous/arithmetic/CstCArExpression.java

```diff
@@ -0,0 +1,85 @@
+package org.chocosolver.solver.expression.continuous.arithmetic;
```

Comment:
Why this new classe is needed?
I think everything can be moved to `BiCArExpression`.